docs/pages/database-access/reference/cli.mdx

```diff
@@ -38,6 +38,53 @@ $ teleport db start \
 | `--gcp-project-id` | (Only for Cloud SQL) GCP Cloud SQL project identifier. |
 | `--gcp-instance-id` | (Only for Cloud SQL) GCP Cloud SQL instance identifier.|
 
+## teleport db configure create
+
+Creates a sample Database Service configuration.
+
+```code
+$ teleport db configure create --rds-discovery=us-west-1 --rds-discovery=us-west-2
+$ teleport db configure create \
+  --token=/tmp/token \
+  --proxy=proxy.example.com:3080 \
+  --name=example \
+  --protocol=postgres \
+  --uri=postgres://postgres.example.com:5432 \
+  --labels=env=prod
+```
+
+| Flag | Description |
+| - | - |
+| `--proxy` | Teleport proxy address to connect to [0.0.0.0:3080]. |
```

Comment:
Here and elsewhere, I think we should make the default values more explicit, e.g., say "The default is <value>" rather than using square brackets.